lib/resources/test/cypress/integration/app.e2e.ts

```diff
@@ -0,0 +1,13 @@
+describe('Aurelia skeleton app', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should load the page and display the initial page title', () => {
+    cy.title().should('equal', 'Aurelia Navigation Skeleton');
```

Comment:
Probably just remove the title test.

I just found out not all our skeleton has the same page title, which is largely our fault. :-)

Reply:
Removed the title test from default setup, left it in place for the navigation variant.